### PR TITLE
NIT: change master to main

### PR DESCRIPTION
**Pull request by MoLow**

### CODE REVIEW

**Review by BridgeAR — APPROVED** [review]

**Comment by BridgeAR** [comment]
Ideally, the commit message could be changed to e.g., `doc: update main branch name`.

**Review by mcollina — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -175,5 +175,5 @@ charter.
 
 [Node.js YouTube channel]: https://www.youtube.com/c/nodejs+foundation/
 [TSC Charter]: https://github.com/nodejs/TSC/blob/master/TSC-Charter.md
-[deps]: https://github.com/nodejs/node/tree/master/deps
-[tools]: https://github.com/nodejs/node/tree/master/tools
+[deps]: https://github.com/nodejs/node/tree/main/deps
+[tools]: https://github.com/nodejs/node/tree/main/tools
```
Comment (aduh95) [inline README.md:179]: Shouldn't we be using relative links instead? That would work on any branch and even locally.
```suggestion
[deps]: ./deps/
[tools]: ./tools/
```

**Comment by MoLow** [comment]
> Ideally, the commit message could be changed to e.g., `doc: update main branch name`.

done. If anyone can merge this

**Review by aduh95 — APPROVED** [review]

**Pull request merged**